tests/common.go

```diff
@@ -2857,7 +2857,7 @@ func CreateSourceAndDestClusters(cloudCred, orgID string) {
 		expect(err).NotTo(haveOccurred(),
 			fmt.Sprintf("Failed to get kubeconfig path for destination cluster. Error: [%v]", err))
 		logrus.Debugf("Save cluster %s kubeconfig to %s", destinationClusterName, dstClusterConfigPath)
-		CreateCluster(destinationClusterName, cloudCred, dstClusterConfigPath, orgID)
+		CreateCluster(destinationClusterName, dstClusterConfigPath, orgID, cloud_name, uid)
```

Comment:
After create cluster are you doing any verifications that cluster is created successfully

Reply:
Added the verification